src/tjenesteomrader/index.tsx

```diff
@@ -0,0 +1,66 @@
+import Layout from 'src/layout';
+
+import digitalTjenesteBlob from './images/indexBlobs/digitalTjeneste.svg';
+import serviceBlob from './images/indexBlobs/strategi.svg';
+import datadrevet from './images/indexBlobs/datadrevet.svg';
+import kultur from './images/indexBlobs/kultur.svg';
+import Entrance from './components/entrance/entrance';
+
+const Tjenesteområder = () => {
+  return (
+    <Layout>
+      <div>
+        <h2 style={{ marginBottom: '2.125rem' }}>Våre tjenesteområder</h2>
+        <p style={{ marginBottom: '2.125rem' }}>
```

Comment:
Eget stylesheet kanskje. Legge inn flex og bruke gap isteden for å ha margin bottom på Entrance komponenten